manuscripts/esquery.py

```diff
@@ -387,42 +296,37 @@ def get_count(cls, date_field=None, start=None, end=None, filters=None):
         :param start: date from which to start counting
         :param end: date until which to count items
         :param filters: dict with the filters to be applied
-        :return: a string with the DSL query
+        :return: a DSL query with size parameter
         """
         """ Total number of items """
         query_basic = cls.__get_query_basic(date_field=date_field,
                                             start=start, end=end,
                                             filters=filters)
-        query = """
-            {
-              "size": 0,
-              %s
-              }
-        """ % (query_basic)
-
+        # size=0 gives only the count and not the hits
+        query = query_basic.extra(size=0)
         return query
 
     @classmethod
     def get_agg(cls, field=None, date_field=None, start=None, end=None,
```

Comment:
As commented above, in other comments, try to clean up the profile a bit, so that for example fields is a list, dates are datetime or similar, etc. Please remember to do so for all methods.

Reply:
Okay